Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nginx:latest
+FROM nginx:1.26.1
```

Comment:
Would be very happy to discuss a sustainable process here, but I've been fixing the version of nginx to use at release time, so we are regularly rebuilding snaphot images (tagged with `next`) using the corresponding `latest` version of nginx.  

Note that there is a scheduled weekly rebuild of our base images in our TC to ensure we pick up background updates to the source images and re-publish regularly under that next tag.  Toolbox uses the `next` tag deliberately so it can act as a canary.

Great to get updated to current stable nginx - I just wanted to clarify how that was working - assume we would cut a release shortly with this change and then can revert.